packages/toml/src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Taplo } from '@taplo/lib'
+import type { Plugin } from 'prettier'
+
+import { defaultOptions, options } from './options'
+import type { PrettierOptions, TaploOptions } from './types'
+
+const PLUGIN_NAME = 'toml'
+
+let taplo: Taplo | null = null
+
+async function format(code: string, options: TaploOptions) {
+  if (!taplo) {
+    taplo = await Taplo.initialize()
+  }
+
+  return taplo.format(code, { options })
+}
+
+function removeBeginningTrailingNewline(code: string) {
+  code = code.replace(/^(?:\r?\n)+/, '')
+  code = code.replace(/(?:\r?\n)+$/, '')
+
+  return code
+}
+
+const TomlPlugin: Plugin<string> = {
+  languages: [
+    {
+      name: PLUGIN_NAME,
+      parsers: [PLUGIN_NAME],
+    },
+  ],
+  parsers: {
+    [PLUGIN_NAME]: {
+      async parse(code: string, options: PrettierOptions) {
+        const indentString = options.useTabs
+          ? '\t'
+          : ' '.repeat(options.tabWidth)
+
+        const taploOptions: TaploOptions = {
+          ...options,
+          columnWidth: options.printWidth,
+          indentString,
+          trailingNewline: true,
+          arrayTrailingComma: options.trailingComma !== 'none',
+          allowedBlankLines: 1,
```

Comment:
Why this option is not exposed?

Reply:
@JounQin Prettier itself only allows up to one newline.

Reply:
Personally I love and hate prettier's option philosophy, while we've declare that this plugin `may not match prettier's format sometimes`, and it's just a simple option which is already supported by `taplo`, so I'd like to expose this option at the same time. How do you think? Don't be shy to disagree.

Reply:
I'm open to expose this option, let's make it default to `1` but allow users customize it :)